src/client.py

```diff
@@ -48,11 +49,6 @@ def start(self):
                 logger.debug("Main renderer waiting for game starts.")
                 self.state.game_starts.acquire()
```

Comment:
you want to modify the conditional check to be `if scene == SceneState.PLAYER_WAIT:`